src/plotReducer.ts

```diff
@@ -18,15 +18,24 @@ export function reducer(state: State, action: ReducerActions) {
       let currentAxis = state.axis[id];
       if (currentAxis) {
         validatePosition(currentAxis.position, position, id);
-        state.axis[id] = { ...currentAxis, position, ...values };
+        state.axis[id] = {
+          ...currentAxis,
+          position,
+          ...values,
```

Comment:
So this makes a merge of the configuration of both axes? We can merge this now if you want but I consider that a bug.

Reply:
This was something that I wasn't sure if we want them separately and it was the current behavior, if this is going to be saved separately then how we decide the min value in a case like this? Should we throw an error?
```tsx
<Axis min={0} id="x" position="top" />
<Axis min={-1} id="x" position="bottom" />
```

Reply:
It's a good question, and I think we need to answer it anyway, even with the merge behavior.

Reply:
The merge behavior takes always the last value, so in the end, we could keep doing that for the moment for all the properties that modify the series context

Reply:
Something we could do is have another component (for example named `<SideAxis>`) which can only define some of the axis properties and can only be rendered if a "main" axis with the same ID exists.

Reply:
I like that idea, let me think on the API